src/ExpressionEvaluator/CSharp/Source/ExpressionCompiler/CompilationContext.cs

```diff
@@ -296,17 +288,25 @@ private static string GetNextMethodName(ArrayBuilder<MethodSymbol> builder)
                     if (!argumentsOnly)
                     {
                         // Pseudo-variables: $exception, $ReturnValue, etc.
-                        if (aliases.Count > 0)
+                        var aliasNames = PooledHashSet<string>.GetInstance(); // To de-dup $ReturnValue/$returnvalue, etc.
```

Comment:
Can we avoid creating duplicates for $ReturnValue in the first place? (Could PlaceholderLocalBinder allow "$returnvalue" explicitly? Or is it really necessary to support "$returnvalue", especially after "$ReturnValue" is included in the Locals window?)